src/manageiq_client/api.py

```diff
@@ -393,12 +406,14 @@ def _load_data(self):
         else:  # Malformed
             raise ValueError("Malformed data: {!r}".format(self._data))
 
-    def reload(self, expand=None, get=True, attributes=None):
+    def reload(self, expand=None, get=True, attributes='_default'):
         kwargs = {}
         if expand:
             if isinstance(expand, (list, tuple)):
                 expand = ",".join(map(str, expand))
             kwargs.update(expand=expand)
+        if attributes == '_default':
+            attributes = self._attributes
```

Comment:
Consider using empty list/tuple to denote no attributes.
Then you also could (but debatable if should) keep None to mean "whatever I was constructed with".
(nit: `_default` string has special meaning in position where strings are acceptable; ideally would use a global distinct sentinel eg. `DEFAULT_ATTRIBUTES = object()`)

P.S. should `expand` also get similar treatment?  IIUC, currently if you load collection with expand=True, take an Entity out of it, and reload() that entity, the expand will be forgotten.